pkg/chartmuseum/server/multitenant/server.go

```diff
@@ -187,3 +200,21 @@ func (server *MultiTenantServer) genIndex() {
 	echo(string(entry.RepoIndex.Raw[:]))
 	exit(0)
 }
+
+func (server *MultiTenantServer) CheckTemplateFilesExist(path string, logger *cm_logger.Logger) bool {
+	// check if template file exists
+	webTemplateFolder, err := os.Open(path)
+	if err != nil {
+		return false
+	}
+	templates, err := webTemplateFolder.Readdir(0)
```

Comment:
Put the error check here ? But great choose for using Readdir , should make sense here .

Reply:
ups! the error check was under the `range`. Moved it up. Thanks for spotting!